### PR TITLE
Add seeds for users, userproducts, subsriptions, crateproducts

### LINKED ISSUE #50: Add seeds
- [x] Add seeds for user
- [x] Add seeds for crateproducts
- [x] Add seeds for userproducts
- [x] Add seeds for subscriptions

### PR DESCRIPTION
**Pull request by catherinemdean15**
Co-authored-by: Isabelle Villasenor <45639675+isabellevillasenor@users.noreply.github.com>

### Description
Adds seeds! 

#### Closes Issues 
#50 

Be sure to seed your database `sequelize db:seed`

### CODE REVIEW

File: code/api/src/seeders/1-user.js [hunk 1]
```diff
@@ -22,6 +22,30 @@ module.exports = {
         role: params.user.roles.user,
         createdAt: new Date(),
         updatedAt: new Date()
+      },
+      {
+        name: 'Susie Shopper',
```
Comment (gaj23) [inline code/api/src/seeders/1-user.js:27]: Great! We'll be able to work through more unit testing with this. Thank you.

File: code/api/src/seeders/4-userproducts.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+const bcrypt = require('bcrypt');
+const config = require('../config/server.json');
+const params = require('../config/params.json');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('userproducts', [
+      {
+        userId: 2,
```
Comment (gaj23) [inline code/api/src/seeders/4-userproducts.js:11]: This is helpful to see and visualize. I appreciate the clarity the API contract has, but this helps even more.

File: code/api/src/seeders/6-crateproducts.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+const bcrypt = require('bcrypt');
+const config = require('../config/server.json');
+const params = require('../config/params.json');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('crateproducts', [
+      {
+        crateId: 1,
```
Comment (gaj23) [inline code/api/src/seeders/6-crateproducts.js:11]: To clarify, this represents products that exists in a specific crate, yes?

**Review by gaj23 — APPROVED** [review]
I'm excited to see this data rendered.

File: code/api/src/seeders/6-crateproducts.js (hunk 3 above)
Comment (catherinemdean15) [reply code/api/src/seeders/6-crateproducts.js:11]: Yes! The data in this table connects a crate with a product.

**Review by tashiad — APPROVED** [review]

**Pull request merged**